helm-chart/templates/cvat_backend/worker_analyticsreports/deployment.yml

```diff
@@ -61,6 +61,26 @@ spec:
           {{- with concat .Values.cvat.backend.additionalEnv $localValues.additionalEnv }}
           {{- toYaml . | nindent 10 }}
           {{- end }}
+          {{- if .Values.cvat.backend.worker.readinessProbe.enabled }}
+          readinessProbe:
```

Comment:
I don't think it makes sense to define a readiness probe for a worker process. According to the [documentation](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/),

> The kubelet uses readiness probes to know when a container is ready to start accepting traffic. [...] One use of this signal is to control which Pods are used as backends for Services.

Workers do not accept traffic or provide a service, so the concept of readiness is meaningless for them.